docs/README.md

```diff
@@ -621,7 +621,7 @@ The `Link` React component is responsible for rendering an `a` HTML element that
 | :------------- |:-------------| :-----|:-----|
 | `page`     | `string`  | The name of the page that the user will be redirected to. Maps to a `blocks.json` block (e.g., `'store.product'`)||
 | `to`     | `string`    |The URL of the page that the user will be redirected to (e.g., `/shirt/p?skuId=1`). Notice that `to` is an **alternative** to `page` and it contains the whole URL instead of the page name. | |
-| `params` | `object`      | The `param` values of the page path in a key-value format (e.g, `{slug: 'shirt'}`). | `{}`|
+| `params` | `object`      | The `param` values of the page path in a key-value format (e.g, `{slug: 'shirt'}`). Params that starts with `__` are not considered on path transformations, and can be generaly be used as an alternative to query params | `{}`|
```

Comment:
Can you show me an example of this change? I have no context about it.

Reply:
The `__listName` property I'm sending from Product Summary is an example of that. Do you think I should add or reference it to the docs?

Reply:
I think so! An example is always great 😄